src/main/scala/com/typesafe/conductr/client/ConductRController.scala

```diff
@@ -26,11 +26,15 @@ import play.api.libs.json.Reads
 
 object ConductRController {
 
-  private[this] val BundlePath = "(.*?)(-[a-fA-F0-9]{0,64})?\\.zip".r
+  /*
+   * Supported ConductR Control Protocol versions.
+   */
+  final val ApiVersionV10 = "1.0"
```

Comment:
What about putting that into a separate scala class / file under the `client` package:

```
object ApiVersion {
  trait Version
  case class V10 extends Version
  case class V11 extends Version
}
```